e2e/integration/apicatalogtests.js

```diff
@@ -0,0 +1,54 @@
+/// <reference types="Cypress" />
+Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false;
+} )
+context( 'Home Page Tests', () => {
+    before( () => {
+        cy.visit( Cypress.env( 'QA_HOST' ) +'developers/api-catalog');
```

Comment:
As i see in before PR also Why this env change do we have to update jenkins also?

Reply:
Nope